.gitignore

```diff
@@ -9,5 +9,5 @@
 node_modules
 
 #bundles
-dist/vendor.*
+dist
```

Comment:
I'd like to draw attention that this change will break work of release script

It's neccessary order when you merge the pr:

1) add dist to .gitignore in master (merge pr)
2) manually merge master to release
3) remove dist from .gitignore in release
4) manually push release